### DISCUSSION TITLE
G-force request

### DISCUSSION THREAD

**Discussion by w-i-z-a-r-d** [theOehrly/Fast-F1#147] [Q&A] [answered]
Is there a way to retrieve the g-force on the driver/car as it shows on the telemetry on screen?

**Comment by markste-in** [comment]
You can calculate it from the speed data but remember... sampling rate is just 4.x Hz. Don't expect overly high accuracy

**Reply by TerminatorPlati** [reply]
Can you provide an example?
Thanks

**Comment by siddharth-tewari** [comment, marked as answer]
You can get an approximation of the g-force values. Keep in mind though that peak g-forces usually only occur for very short amounts of time. Therefore, you could only calculate g-force of long sweeping corners and similar accurately.

_Originally posted by @theOehrly in https://github.com/theOehrly/Fast-F1/discussions/96#discussioncomment-2118096_

This was the reply of @theOehrly on same question by me.
